### PR TITLE
add real action instructions to receipting tests

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Output FieldInstructions

# What has changed
Checks for correct field instructions for receipting

# How to test?
With case-processor and fieldwork-adapter from the ticket run the tests, they should pass

# Links
https://trello.com/b/yarvSxGu/census-2021-response-management

### CODE REVIEW

File: acceptance_tests/features/receipt.feature [hunk 1]
```diff
@@ -6,19 +6,20 @@ Feature: Case processor handles receipt message from pubsub service
     When the receipt msg is put on the GCP pubsub
     Then a uac_updated msg is emitted with active set to false for the receipted qid
     And the correct events are logged for "[<loaded case events>]" and "[<individual case events>]"
-    And if the "<instruction>" is not NONE, a case updated event with actual responses is "<increment>" and receipted "<receipt>" for case type "<case type>"
+    And if the case is updated by an "<increment>" or by a "<receipt>" then there should be a case updated message of "<case type>"
```
Comment (AdamHawtin) [inline acceptance_tests/features/receipt.feature]: This reads fine in the feature file but the values are substituted in in the logs so what you see logged in a test run is 
`And if the case is updated by an "False" or by a "True" then there should be a case updated message of "SPG"` 

Maybe something like 
` And if the actual response count is incremented (<increment>) or the case is marked receipted (<receipt>) then there should be a case updated message of case type "<case type>"` 
To keep the log output readable?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: acceptance_tests/features/receipt.feature (hunk 1 above)
Comment (LukeLoze1) [reply acceptance_tests/features/receipt.feature]: yeah could do,  wasn't sure either way as was also trying to keep lines a bit shorter, but I'll change it

**Review by AdamHawtin — APPROVED** [review]
👍

File: acceptance_tests/features/receipt.feature [hunk 2]
```diff
@@ -2,23 +2,24 @@ Feature: Case processor handles receipt message from pubsub service
...
       | SPG       | E             | Ind      | False     | False   | NONE        | sample_1_ni_SPG_estab.csv     | N       | RESPONSE_RECEIVED,RM_UAC_CREATED,FULFILMENT_REQUESTED,SAMPLE_LOADED                     |                                  |
+      | SPG       | U             | HH       | False     | True    | CLOSE       | sample_1_english_SPG_unit.csv | E       | SAMPLE_LOADED,RESPONSE_RECEIVED                                                         |                                  |
       | SPG       | U             | Ind      | False     | False   | NONE        | sample_1_english_SPG_unit.csv | E       | RESPONSE_RECEIVED,RM_UAC_CREATED,FULFILMENT_REQUESTED,SAMPLE_LOADED                     |                                  |
       | SPG       | U             | Cont     | False     | False   | NONE        | sample_1_english_SPG_unit.csv | E       | RESPONSE_RECEIVED,RM_UAC_CREATED,PRINT_CASE_SELECTED,FULFILMENT_REQUESTED,SAMPLE_LOADED |                                  |
```
Comment (AdamHawtin) [inline acceptance_tests/features/receipt.feature:24]: From this scenario, I can't work out why some of the examples expect `PRINT_CASE_SELECTED` or `FULFILMENT_REQUESTED` events against the case?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: acceptance_tests/features/receipt.feature (hunk 2 above)
Comment (AdamHawtin) [reply acceptance_tests/features/receipt.feature:24]: Found it, those print or fulfilment requests are to generate the QID type required by that example row, makes sense.

**Review by AdamHawtin — APPROVED** [review]

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Pull request merged**